tests/end_to_end/test_vulnerability_detector/test_vulnerability_detector.py

```diff
@@ -58,7 +55,7 @@
 from wazuh_testing.tools.configuration import load_configuration_template
 from wazuh_testing.tools.system import HostManager
 from wazuh_testing.end_to_end.remote_operations_handler import launch_parallel_operations
-from wazuh_testing.end_to_end.vulnerability_detector import get_alerts_by_agent, get_indexed_vulnerabilities_by_agent
```

Comment:
Now that these functions are not used, are they needed in `deps/wazuh_testing/wazuh_testing/end_to_end/vulnerability_detector.py`?

Reply:
Done 4c08533990e882e0adf9a4f800714e71a013b3ae